chore: update readme (#17)
evva-sfw/nest-mqtt: README.md

## Usage

Import the module wherever needed and inject the `MqttService` into your service for use.

```ts
@Module({
  imports: [MqttModule],
})
export class ExampleModule {
  constructor(private readonly mqttService: MqttService) {}
  
  public async example() {
    try {
      await this.mqttService.connect({
        host: '127.0.0.1',
        protocol: 'mqtt',
        port: 1883,
        username: 'test',
        password: 'test',
        autoSubscribe: true
      })
    } catch (e) {
      console.log(`Failed to connect: ${e}`)
    }
  }

  @Subscribe('exampleTopic')
  public onExampleTopicEvent(@Payload() payload: object) {
    console.log(payload)
  }
}
```

> > Tip: Use the `autoSubscribe` flag to automatically explore and subscribe to configured topics.

Reviewer: How about this to make it easier to understand on what part the autoSubscribe depends?
```suggestion
> Tip: Use the `autoSubscribe` flag to automatically subscribe to all @Subscribe() decorators.
```
